cdap-support-bundle/src/main/java/io/cdap/cdap/support/internal/app/services/SupportBundleService.java

```diff
@@ -111,21 +107,27 @@ public String generateSupportBundle(SupportBundleConfiguration supportBundleConf
       .build();
     addToStatus(supportBundleStatus, uuidPath.getPath());
 
-    SupportBundleJob supportBundleJob =
-      new SupportBundleJob(supportBundleTaskFactories, executorService, cConf, supportBundleStatus);
-    SupportBundleTaskConfiguration supportBundleTaskConfiguration =
-      new SupportBundleTaskConfiguration(supportBundleConfiguration, uuid, uuidPath, namespaces, supportBundleJob);
-
+    ExecutorService executor = Executors.newFixedThreadPool(cConf.getInt(Constants.SupportBundle.MAX_THREADS),
```

Comment:
Are we creating a new executor service every time somebody calls the generate support bundle endpoint? Shouldn't the executor pool be created during startup?

Reply:
Ah please disregard, this is fixed in the most recent push.